packages/twenty-front/src/modules/activities/files/components/DropZone.tsx

```diff
@@ -0,0 +1,121 @@
+import { useDropzone } from 'react-dropzone';
+import styled from '@emotion/styled';
+
+import { useSpreadsheetImportInternal } from '@/spreadsheet-import/hooks/useSpreadsheetImportInternal';
+import { IconUpload } from '@/ui/display/icon';
+
+const StyledContainer = styled.div`
+  align-items: center;
+  background-position:
+    0 0,
+    0 0,
+    100% 0,
+    0 100%;
+  background-repeat: no-repeat;
+  background-size:
+    2px 100%,
+    100% 2px,
+    2px 100%,
+    100% 2px;
+  border-radius: ${({ theme }) => theme.border.radius.md};
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  justify-content: center;
+  position: relative;
+  height: 100%;
+  border: ${({ theme }) => `2px dashed ${theme.border.color.strong}`};
+`;
+
+const StyledOverlay = styled.div`
+  background: ${({ theme }) => theme.background.secondary};
+  border-radius: ${({ theme }) => theme.border.radius.sm};
+  bottom: 0px;
+  left: 0px;
+  position: absolute;
+  right: 0px;
+  top: 0px;
+`;
+
+const StyledUploadDragTitle = styled.div`
+  color: ${({ theme }) => theme.font.color.primary};
+  font-size: ${({ theme }) => theme.font.size.lg};
+  font-weight: ${({ theme }) => theme.font.weight.medium};
+  line-height: ${({ theme }) => theme.text.lineHeight.md};
+  margin-bottom: 8px;
+  z-index: 1;
+`;
+
+const StyledUploadDragSubTitle = styled.div`
+  color: ${({ theme }) => theme.font.color.tertiary};
+  font-size: ${({ theme }) => theme.font.size.md};
+  font-weight: ${({ theme }) => theme.font.weight.regular};
+  line-height: ${({ theme }) => theme.text.lineHeight.md};
+  z-index: 1;
+`;
+
+const StyledUploadDragImage = styled.div`
+  margin-bottom: 12px;
+  z-index: 1;
+
+  svg {
+    color: ${({ theme }) => theme.font.color.tertiary};
+    font-size: ${({ theme }) => theme.font.size.md};
+  }
+`;
```

Comment:
Usually to style an icon, we wrap the Icon with `styled`: `styled(IconUpload)`. Then instead of styling `svg`, we can style it like this:

```
const StyledUploadIcon = styled(IconUpload)`
  color: ${({ theme }) => theme.font.color.tertiary};
  margin-bottom: ${({ theme }) => theme.spacing(3)};
`;

// ...

const theme = useTheme();

// ...

<StyledUploadIcon
  stroke={theme.icon.stroke.sm}
  size={theme.icon.size.lg}
/>

Reply:
Also, @Bonapara: the icon size in the Figma is 24px, but in the theme we only have these icon sizes:
```
size: {
  sm: 14,
  md: 16,
  lg: 20,
  xl: 40,
},
```
Here I advised @jeet1desai to use the `lg` size as it is the closest. Is it ok?

Reply:
Let's use a 20px (lg) bold then. Thanks @thaisguigon